src/runtime/auto-prompt-manager.js

```diff
@@ -419,14 +419,32 @@ export class AutoPromptManager {
       autoPromptType === AutoPromptType.CONTRIBUTION ||
       autoPromptType === AutoPromptType.CONTRIBUTION_LARGE
     ) {
-      if (!dismissedPrompts) {
+      const surveyTriggeringPriorityEnabled = await this.isExperimentEnabled_(
```

Comment:
I'm not quite clear what `surveyTriggeringPriorityEnabled` means. Could you describe it a bit?

Reply:
`surveyTriggeringPriority` refers to the experiment, perhaps it will be more clear if I name it as such? `surveyTrigginerPriorityExperimentIsEnabled`

Reply:
Would it make sense to call it `preferSurveyOverContributionPrompt`?